Automattic/wp-calypso
# State: Update preferences to use wpcom.req directly

#### Changes proposed in this Pull Request

This PR updates the preferences actions to use `wpcom.req` directly rather than having to use the separate wpcom-undocumented methods. It also removes those unused undocumented methods.

Part of the effort to remove `wpcom-undocumented` and make the state more modular and packageable.

#### Testing instructions

* Open any page in Calypso.
* Verify preferences are still loaded correctly by rolling over the environment badge on the bottom right.
* Change a preference, verify it's still saved correctly.
